CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [2.1.26] - 2022-06-09
+### Changed
+- Stop always allowing today's date to be picked
+
```

Comment:
Need need to add a new version, you can add your change to `2.1.25`, it hasn't been released yet